lib/go-tc/traffic_router.go

```diff
@@ -136,6 +136,7 @@ type TrafficServer struct {
 	CacheGroup       string              `json:"cacheGroup"`
 	IP6              string              `json:"ip6"`
 	Port             int                 `json:"port"`
+	HTTPSPort        int                 `json:"httpsPort,omitempty"`
```

Comment:
This is the subject of some debate, so I'd appreciate other people chiming in, but personally I feel that fields that are `NULL` should marshal as `null` and not `undefined` - it keeps the structure more deterministic and easier to deal with.

Reply:
I added `omitempty` because this variable is being used internally by the Monitor, but isn't in the `monitoring.json` endpoint. This preserves the existing API, not including `omitempty` would change the API.

Reply:
if it's not exposed then it's a non-issue - which is fortunate because that argument could go back and forth for a while if enough of the right people saw that comment.